app/models/hackbot/dispatcher.rb

```diff
@@ -27,8 +27,9 @@ def run_convo(convo, event)
     end
 
     def convos_needing_handling(event)
-      Hackbot::Conversation.where.not(state: :finish)
-                           .select { |c| c.part_of_convo? event }
+      Hackbot::Conversation.where.not(state: :finish) .select do |c|
+        c.part_of_convo? event
+      end
```

Comment:
Hmm, it shouldn't say that – the original was 65 characters long and the limit's 80. Can you revert this part of the change?